src/components/Budget/CreateBudgetDialog/CreateBudgetDialog.tsx

```diff
@@ -12,7 +12,9 @@ import {
 } from "../../../reducers/notificationReducer";
 import dataInputFields from "../../../utils/inputFields.json";
 import { IInputField } from "../../../models";
-import CommonDialog from "../CommonDialog";
+import CommonDialog from "../../Dasboard/CommonDialog";
+import CommonInputForm from "../../Forms/CommonInputForm/CommonInputForm";
```

Comment:
The naming convention doesn't seem right. `CommonInputForm` could be in the specific directory of inputs.  
also in `CommonInputForm` directory, we can have an `index.js` to default export `CommonInputForm` so the import statement will look like 
`	import CommonInputForm from "../../Forms/CommonInputForm";`